ldt/make/Filepath

```diff
@@ -1 +1 @@
+dirs := .  ../main ../core ../interp ../plugins ../lib/bil ../lib/xmrg ../params/mask ../params/landcover ../params/soils ../params/topo ../params/albedo ../params/gfrac ../params/LAISAI ../params/slopetype ../params/tbot ../params/pet ../params/climateparms ../params/crop_parms ../params/irrigation ../params/catchment ../params/CLM2 ../params/GeoWRSI ../params/FLAKE ../params/HYMAP ../params/JULES ../params/Mosaic ../params/Noah ../params/AquaCrop ../params/RDHM ../params/RUC ../params/SACHTET ../params/Snow17 ../params/SnowModel ../params/SiB2 ../params/VIC ../params/glacier ../runmodes/ANNproc ../runmodes/DApreproc ../runmodes/LSMparamproc ../runmodes/EnsRstpreproc ../runmodes/NUWRFpreproc ../runmodes/Metforcproc ../runmodes/MetTimeDScale ../runmodes/climoRstproc ../runmodes/rstTransformProc ../runmodes/StatDscaleMetForc ../runmodes/OPTUEproc/ ../domains/latlon ../domains/gaussian ../domains/hrap ../domains/lambert ../domains/polar ../domains/merc ../domains/easev2 ../DAobs/MCD15A2H_LAI ../DAobs/LISlsmSMobs ../DAobs/LISlsmPrecipobs ../DAobs/LISlsmTEFFobs ../DAobs/synthetic_sm ../DAobs/NASA_AMSRE_sm ../DAobs/LPRM_AMSREsm ../DAobs/LPRMvod ../DAobs/ESACCI_sm ../DAobs/WindSat_sm ../DAobs/GRACE_tws ../DAobs/GRACEQL_tws ../DAobs/SMOPS ../DAobs/ASCAT_TUW ../DAobs/SMOS_L2sm ../DAobs/SMOS_NESDIS ../DAobs/GCOMW_AMSR2L3snd ../DAobs/GCOMW_AMSR2L3sm ../DAobs/Aquarius_L2sm ../DAobs/simGRACE_JPL ../DAobs/SMMR_SNWD ../DAobs/SSMI_SNWD ../DAobs/ANSA_SNWD ../DAobs/NASA_SMAPsm ../DAobs/SMOS_NRTNN_L2sm ../DAobs/NASA_SMAPvod ../DAobs/GLASSlai ../DAobs/THySM ../DAobs/GEOS_FP_TEFFobs ../DAobs/SMAP_E_OPLsm ../DAobs/VIIRS_GVF ../DAobs/CDFS_GVF ../RESTRICTED/usaf/usaf ../metforcing/agrradps ../metforcing/cmap ../metforcing/cmorph ../metforcing/ecmwf ../metforcing/gdas ../metforcing/geos5fcst ../metforcing/gfs ../metforcing/gldas ../metforcing/gswp1 ../metforcing/gswp2 ../metforcing/merra2 ../metforcing/era5 ../metforcing/nam242 ../metforcing/narr ../metforcing/princeton ../metforcing/nldas2 ../metforcing/WRFoutv2 ../metforcing/WRFAKdom ../metforcing/stg2 ../metforcing/stg4 ../metforcing/3B42RTV7 ../metforcing/3B42V6 ../metforcing/3B42V7 ../metforcing/RFE2Daily ../metforcing/RFE2gdas ../metforcing/chirps2 ../MetforcScale ../ANNdata/LISlsmSM ../ANNdata/synthetic_sm ../ANNdata/LPRM_AMSREsm ../ANNdata/MODIS_LST ../ANNdata/MOD10A1 ../ANNdata/GHCNsnwd ../ANNdata/GCOMW_AMSR2_TB ../statDscale/BayesianMerging ../statDscale/Climo ../params/CLM45 ../USAFSI ../SMAP_E_OPL ../runmodes/USAFSI ../runmodes/LISHydropreproc ../runmodes/SMAP_E_OPL ../params/Crocus ../runmodes/obsSim ../obsSim/NatureRun/LISout/ ../obsSim/OSSEmask/LISout/ ../obsSim/OSSEmask/AMSR2/ ../obsSim/OSSEmask/TSMM/ ../obsSim/OSSEmask/MODIS/ ../obsSim/OSSEmask/Sentinel1A/
```

Comment:
I noticed one new issue. Filepath is generated during compilation. The file should not be added itself.

Reply:
Sorry. My bad. It's strange that on the nasa repo it's not part of the lis make folder but it's- indeed part of the ldt make folder. Normally, it's not included I think since it's generated during compilation...